test/unit/browser/System.js

```diff
@@ -83,7 +83,7 @@ suite('System.js', function() {
 
   test('System.set', function() {
     var store = $traceurRuntime.ModuleStore;
-    var polyfills = store.getForTesting('src/runtime/polyfills/polyfills.js');
+    var polyfills = store.get(System.version + '/src/runtime/polyfills/polyfills.js');
```

Comment:
We could add 'test/unit/browser/*.js' to the patterns, then remove it in NodeTraceurTestRunner. 

I should have done the `/node/` exclusion in BrowserTraceurTestRunner.